### PR TITLE
Clam 1493 Windows services

### PR DESCRIPTION
**Pull request by kang-grace**
Adds Windows service options for freshclam and clamd.

### CODE REVIEW

File: unit_tests/freshclam_test.py [hunk 1]
```diff
@@ -263,8 +263,8 @@ def test_freshclam_05_cdiff_update(self):
         # start with this CVD
         shutil.copy(TC.path_source / 'unit_tests' / 'test_db' / 'test-1.cvd', TC.path_db / 'test.cvd')
 
-        # update to this CVD
```
Comment (micahsnyder) [inline unit_tests/freshclam_test.py:266]: It appears that your commit somehow includes bits of a previous commit, I think as a result of a merge.  Can I trouble you to remove the changes to freshclam_test.py?

**Comment by micahsnyder** [comment]
@kang-grace we should add a description of this new feature in the `NEWS.md` file for the 0.104 release, and credit @sherpya for his work in the news burb and also in the Acknowlegdements section.

@sherpya if you have any concerns about how we're importing your work, your review will be appreciated.

File: unit_tests/freshclam_test.py (hunk 1 above)
Comment (kang-grace) [reply unit_tests/freshclam_test.py]: Oops - should be fixed now.

**Comment by micahsnyder** [comment]
The Jenkins pipeline looks great.  *Everything* passed. 

Here's the main clamd service test output:
image

And the main freshclam service test output:
image

**Comment by sherpya** [comment]
> @kang-grace we should add a description of this new feature in the `NEWS.md` file for the 0.104 release, and credit @sherpya for his work in the news burb and also in the Acknowlegdements section.
> 
> @sherpya if you have any concerns about how we're importing your work, your review will be appreciated.

looks ok, a see a new flag "service-mode", I was always calling svc_stuff, they seems harmless, anyway better to call only if in service mode

**Comment by micahsnyder** [comment]
Everything look good to me. 

Still seeing some odd failures on windows with:
`Wrong reply for command zMULTISCAN \nonexistentfilename.`
`Wrong reply for command SCAN \nonexistentfilename`
But these aren't your fault. 

I'll squash & merge it now. 
Nice work!

**Pull request merged**